unixfs.go

```diff
@@ -361,6 +361,11 @@ func EmptyDirNode() *dag.ProtoNode {
 	return dag.NodeWithData(FolderPBData())
 }
 
+// EmptyFileNode creates an empty file Protonode.
+func EmptyFileNode() *dag.ProtoNode {
```

Comment:
Does this need to be exported or is it an implementation detail?

Reply:
I'm following the pattern of the existing (public) `EmptyDirNode()` function here.